test/commands/schema/apply.test.js

```diff
@@ -0,0 +1,132 @@
+const Nock = require('@fancy-test/nock').default;
+const { expect, test } = require('@oclif/test');
+const fs = require('fs');
+
+const fancy = test.register('nock', Nock);
+
+describe('schema:apply', () => {
+  let parsedBody;
+
+  before(() => {
+    const forestadminSchema = `{
+      "meta": {
```

Comment:
Did you take it from an actual `foreastadmin-schema.json` ? The keys should be ordered

Reply:
It is probaly from the old version, but anyway, it doesn't change what it is tested.